### DISCUSSION TITLE
motioneye already installed

### DISCUSSION THREAD

**Discussion by pintail120** [DaniW42/motioneye-telegram#35] [Q&A]
Hi 
I have an existing instance of motioneye, how do I install motion-telegram to it ?
when I ssh into the existing motioneye, I am unable to use sudo or apt
I get that you already root and do not need sudo
but I am unclear if this is a feasable way to install motion-telegram in this retrospective way ?
thanks & regards
Gavin

**Comment by BeyTek** [comment]
pareil

**Comment by DaniW42** [comment]
Which OS do you guys use? In theory there should be no functional problems, beside the security concerns, running it as root. You may have to hardcode different paths into the scripts when your motioneye instance dosn't follow the debian/raspbian paths.

**Reply by DaniW42** [reply]
BTW, Sorry @pintail120, I missed your question...
